lib/friendly-errors/formatters/missing-css-file.js

```diff
@@ -22,7 +22,7 @@ function formatErrors(errors) {
         chalk.red('Module build failed: Module not found:')
     );
     for (let error of errors) {
-        messages.push(`${error.file} contains a reference to the file ${error.ref}.`);
+        messages.push(`${error.file} contains a reference to the file "${error.ref}".`);
```

Comment:
should we wrap `error.file` in quotes too for consistency ?

Reply:
Sounds good. I will do a quick check in the other files in the `lib/friendly-errors` directory as well.

Reply:
After some research I noticed that in `lib/friendly-errors/transformers/missing-css-file.js` (among other files) a single quote is used for referencing to paths and modules. I also found another path reference in another formatter and updated it.

Reply:
I'd rather normalize to quotes rather than apostrophes, as that's about an English message, not a code snippet.

"single quote" vs "double quote" is a language abuse coming from programming languages describing things in a weird way. In Unicode, `"` is named `quotation mark` and `'` is named `apostrophe`. Using `'` as a quotation mark is misusing it.

Also, this would be consistent with the formatting used in error messages in Symfony itself.

Reply:
You are absolutely correct, these are English sentences and should be treated as such. I've refactored all to use quotation marks as you suggested.

I skipped the file I based it off before (`lib/friendly-errors/transformers/missing-css-file.js`). If I understand the file correctly it transforms an error message that originates outside of this repository, so updating it to use quotation marks would break things.